src/components/app/ProfileSharing.js

```diff
@@ -248,6 +256,12 @@ class ProfileSharingCompositeButton extends PureComponent<
       });
   }
 
+  _toggleShareNetworkUrls() {
+    this.setState(prevState => ({
+      shareNetworkUrls: !prevState.shareNetworkUrls,
```

Comment:
Can we use the checkbox status instead of toggling the state ?
eg:
```javascript
// (notice the change to the method name too)
_onChangeShareNetworkUrls(event: SyntheticEvent<HTMLInputElement>) {
  this.setState({
    shareNetworkUrls: event.currentTarget.checked,
  });
}
```
I find it more "direct". What do you think?

Reply:
Yeah, this looks better. Thanks!